src/Metadata/AmazonMetadataBuilder.php

```diff
@@ -31,12 +40,28 @@ final class AmazonMetadataBuilder implements MetadataBuilderInterface
     public const STORAGE_GLACIER = 'GLACIER';
 
     /**
-     * @var array<string, mixed>
+     * @var array
+     *
+     * @phpstan-var AmazonSettings
      */
     private $settings;
 
     /**
-     * @var string[]
+     * @var MimeTypesInterface
+     */
+    private $mimeTypes;
+
+    /**
+     * @var array
+     *
+     * @phpstan-var array{
```

Comment:
What's the purpose of the private property if you can't change the values ?

Reply:
You suggest to move it to the method as a normal variable?

Reply:
If the value can be changed, the phpdoc is wrong.

If the value cannot be changed, then using this in a private var is kinda useless. We can either using it directly in the method or using a constant for this array (instead of a private var).

Reply:
I will move it to constant

Reply:
Done